src/txn/txn_rollback_to_stable.c

```diff
@@ -72,10 +75,12 @@ __rollback_abort_newer_insert(
   WT_SESSION_IMPL *session, WT_INSERT_HEAD *head, wt_timestamp_t rollback_timestamp)
 {
     WT_INSERT *ins;
+    bool stable_update_found;
 
     WT_SKIP_FOREACH (ins, head)
         if (ins->upd != NULL)
-            __rollback_abort_newer_update(session, ins->upd, rollback_timestamp);
+            __rollback_abort_newer_update(
+              session, ins->upd, rollback_timestamp, &stable_update_found);
```

Comment:
I recommend that when returning a variable in a side-effect, you should always initialize it in the called function (`__rollback_abort_newer_update`), and never set it in the caller. If you do that, it makes it harder to introduce bugs in the future because you can't forget to set it.

Reply:
Fixed.